clang/lib/CodeGen/CGExpr.cpp

```diff
@@ -3615,9 +3616,12 @@ void CodeGenFunction::EmitCheck(
 
```

Comment:
maybe we can keep the patch Driver only, because we need:
passing fractions array into LLVM passes: ubsan and hwasan

Reply:
Done